docs/api/qiskit/release-notes/0.18.md

```diff
@@ -5,6 +5,71 @@ description: New features and bug fixes
 
 # Qiskit 0.18 release notes
 
+## 0.18.3
```

Comment:
Weird that there's no 0.18.1 and 0.18.2 in this file. But that's how it already was. This looks right.

Reply:
Yeah, I noticed too. I checked in the legacy_release_notes.md and there are no 0.18.1 and 0.18.2 either